driver-core/src/main/com/mongodb/internal/operation/CommandOperationHelper.java

```diff
@@ -601,15 +606,16 @@ static void addRetryableWriteErrorLabel(final MongoException exception, final in
         }
     }
 
-    static void logRetryExecute(final RetryState retryState) {
+    static void logRetryExecute(final RetryState retryState, final OperationContext operationContext) {
         if (LOGGER.isDebugEnabled() && !retryState.isFirstAttempt()) {
             String commandDescription = retryState.attachment(AttachmentKeys.commandDescriptionSupplier()).map(Supplier::get).orElse(null);
             Throwable exception = retryState.exception().orElseThrow(Assertions::fail);
             int oneBasedAttempt = retryState.attempt() + 1;
+            long operationId = operationContext.getId();
             LOGGER.debug(commandDescription == null
-                    ? format("Retrying the operation due to the error \"%s\"; attempt #%d", exception, oneBasedAttempt)
-                    : format("Retrying the operation '%s' due to the error \"%s\"; attempt #%d",
-                            commandDescription, exception, oneBasedAttempt));
+                    ? format("Retrying the operation due to the error \"%s\"; attempt #%d; operation ID %s", exception, oneBasedAttempt, operationId)
```

Comment:
Let's make the message a bit more similar in structure/style to what's in https://github.com/mongodb/specifications/blob/master/source/command-logging-and-monitoring/command-logging-and-monitoring.rst#command-started-message.   We typically try to use complete sentences, for instance, rather than weird punctuation.

Reply:
Thanks for the suggestion, i have made the necessary changes to align with the recommended style.